pydis_site/templates/home/timeline.html

```diff
@@ -623,8 +622,8 @@ <h2>Lemon on Talk Python To Me</h2>
         </div>
 
         <div class="cd-timeline__content text-component">
-          <h2>Lemon on Teaching Python</h2>
-          <p class="color-contrast-medium">Lemon goes on the Teaching Python podcast to discuss how the pandemic has
+          <h2>We're on the Teaching Python podcast!</h2>
+          <p class="color-contrast-medium">Lemon joins Sean and Kelly on the Teaching Python podcast to discuss how the pandemic has
```

Comment:
Still lemon on that line?

Reply:
I did this on purpose but I don't know, I guess I'm open to suggestions. Leon looks so weird to me. Nobody calls me that. But yeah maybe I should just let go and change this too.

Reply:
Fixed, and squashed fixes into initial commit.

Reply:
Sweet! I think Leon does make sense here, since lemon doesn’t appear anywhere on the timeline. That said, if you prefer lemon I really don’t mind haha